QUEST.tsv

```diff
@@ -2098,156 +2098,156 @@ QUEST_20150714_002097	But, Raius is the disciple I acknowledge.{nl}His passion t
+QUEST_20150714_002156	$This movement was made very naturally. {nl}If that's not so, there definitely must have been conflicts, and more fables would have been created.
+QUEST_20150714_002157	$I believe the Goddesses erased all memory and records of that time completely. {nl}However, we can't deny the possibility that another reason may exist due to the fact that this legend still remains even with the authority of the great Goddess.
+QUEST_20150714_002158	$And this is a very cautious prediction, but the creatures what we call devils may exist in the underground. {nl}Though that theory is hard to accept when we think about all the facts that are recorded in history and the present.
+QUEST_20150714_002159	$I can smell many Avietes. I heard they're effective… Could you share them with us?
+QUEST_20150714_002160	$Thank you. I will give you as many as you need. {nl}You will be successful.
+QUEST_20150714_002161	$I wanted to eat it…
+QUEST_20150714_002162	$You're not collecting Avietes?
+QUEST_20150714_002163	$I'll give them to you. {nl}You will be successful.
+QUEST_20150714_002164	$Our hope will definitely come true.
+QUEST_20150714_002165	$I trust Horaceus.
+QUEST_20150714_002166	$I'll cheer up and smile until the day comes.
+QUEST_20150714_002167	$I guess I'll take this tombstone rubbing to the person who wants it.
+QUEST_20150714_002168	$Since I've already made the rubbing, I'll give to the person who wants it.
+QUEST_20150714_002169	$Copy of Prediction
+QUEST_20150714_002170	$Death brings more death.
+QUEST_20150714_002171	$I am busy right now. We better do it later.
+QUEST_20150714_002172	$Ah, you're the one Adrius sent. {nl}I've obtained the diluting solution you ordered. {nl}Be careful not to break the glass.
+QUEST_20150714_002173	$One glass of diluting solution will be enough.
+QUEST_20150714_002174	$Epitaph of Juan
+QUEST_20150714_002175	$I lie here without yet creating thirteen thousand pieces of armor. {nl}I hope to finish them beside the Goddesses.
+QUEST_20150714_002176	$I can hear a whisper in my head.
+QUEST_20150714_002177	$Please install the device I gave you in the 2nd district in Crystal Mine.
+QUEST_20150714_002178	$The location is Sidabro Rest Place in Veya Valley. {nl}I hope he's still alive.
+QUEST_20150714_002179	$Klaipeda Market
+QUEST_20150714_002180	$Give me the effigy. I shall calm the spirits.
+QUEST_20150714_002181	$The monsters will feel threatened by the energies that spurt out from the crystals containing my authority and begin their attack.
+QUEST_20150714_002182	$Please protect the crystals so that they may produce my authority and spread it to Sirdgelos Forest.
+QUEST_20150714_002183	$You prepared this, brother? You're truly a great man.
+QUEST_20150714_002184	$I can sense the wrath of the spirits.
+QUEST_20150714_002185	$Incense Burner
+QUEST_20150714_002186	$The scent of the Incense Burner is spreading.
```

Comment:
Should be just "incense" here, it's the incense that has a scent, not the burner. We could rewrite it as 
"The scent of the incense is spreading from the Incense Burner.".